advancedDungeon/src/produsAdvanced/level/AdvancedBerryLevel.java

```diff
@@ -34,7 +34,7 @@
  * @see produsAdvanced.riddles.MyPlayerController
  * @see Berry
  */
-public class AdvancedBerryLevel extends BlocklyLevel {
+public class AdvancedBerryLevel extends DevDungeonLevel {
   private static boolean showMsg = true;
```

Comment:
Siehe Review im ctor. Es macht am meisten Sinn diese static in `DevDungeonLevel` zu verschieben, weil auch dort das Popup angezeigt wird.